oxide-auth-rouille/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "oxide-auth-rouille"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
If the `oxide-auth` dependency is publicly visible, including as trait-`impl`s, then this must be `0.2`. (Obviously this is the case as implementing the traits is half the reason to have this crate).